functions.md

```diff
@@ -0,0 +1,131 @@
+## Function
+A function is a reuseable block of code, that group together a sequence of statement to peform a specific task.
+### Function declaration
+in javaScript they are many ways to declare a function.One way to create a function is by using a `function` declaration. example:
+
+```js
+function teamNames() {
+    console.log('Liverpool', 'Barcelona');
+}
+console.log(teamNames()); // Liverpool, Barcelona
+```
+A function declaration consists of:
+
+- The function keyword.
+- The name of the function, or its identifier, followed by parentheses.
+- A function body, or the block of statements required to perform a specific task, enclosed in the function’s curly brackets, { }.
+
+### Calling a function
+A function is called when `function` name followed by the parentheses is typed. From the example above
+
+```js
+console.log(teamNames());
+```
+called the function inside `teamNames`.
+
+### Parameter and Argument
+parameter allows function accept input(s), and perform a task using the input(s). When declaring a function we can specify a parameter.  We use parameters as placeholders for information that will be passed to the function when it is called. example:
+
+```js
+function aggregateScore(Home, Away) {
+    console.log(Home + Away);
+}
```

Comment:
This is not too important, but I would recommend using `camelCase` for your parameters just as you used for your function name, that is, words start will lower cases, so `Home` and `Away` should be `home` and `away` resp.

Reply:
👍